realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -112,7 +113,6 @@ public final AllTypesColumnInfo clone() {
         proxyState.setConstructionFinished();
     }
 
-    @Override
```

Comment:
Because it is not being emitted by the code, which caused the UT to fail.  Since none of the other overridden methods are annotated, I thought it was easier to remove this one, that to change all of the other ones.